apps/src/templates/testPages/WebLabTest.jsx

```diff
@@ -1,17 +1,21 @@
 import React, {Component} from 'react';
-import {Status} from '../../../src/lib/ui/ValidationStep';
+import ValidationStep, {Status} from '../../../src/lib/ui/ValidationStep';
 import testImageAccess from '../../code-studio/url_test';
 
+const scriptData = document.querySelector('script[data-bramble]');
+const brambleConfig = JSON.parse(scriptData.dataset.bramble);
```

Comment:
our convention is to do this step (grabbing data from the DOM) in the "entrypoint" (`weblab_host/test.js` in this case) and pass this data into the component as props to keep the two decoupled